src/App.tsx

```diff
@@ -15,71 +15,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  setGood = (event: React.MouseEvent<HTMLButtonElement>) => {
+    const goodName = event.currentTarget.closest('tr')
+      ?.querySelector('.is-vcentered')
+      ?.textContent;
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    this.setState({ selectedGood: goodName });
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    if (this.state.selectedGood === goodName) {
+      this.setState({ selectedGood: '' });
+    }
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  removeGood = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  render() {
+    const { selectedGood } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    const goodIsSelectedMessage = (
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood}
+        {' is selected'}
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
try to avoid disabling eslint rules - lets fix them

Reply:
The only way I found to avoid this eslint rule disabling is to put some invisible content in the button tag